src/vanilla/Model/ResponseJv.cs

```diff
@@ -233,13 +270,23 @@ public IEnumerable<string> ImplImports
                 var imports = new List<string>(InterfaceImports);
                 imports.AddRange(BodyWireType.ImportSafe());
                 imports.AddRange(HeaderWireType.ImportSafe());
-                if (this.NeedsConversion && (Body is SequenceType || Headers is SequenceType))
+
+                if (ReturnValueWireType != null)

```

Comment:
I'd encourage single accesses on computed properties that perform traversals. This property is probably not an offender, but the time needed to generate code has been increased substantially by unwitting repeated accesses to properties that perform traversals of the Swagger model.

It's got me considering taking computed properties that do nontrivial things and turning them into methods when I encounter them simply so that people don't use them as though it's just reading a value from a field.